pygtfs/loader.py

```diff
@@ -46,8 +44,7 @@ def overwrite_feed(schedule, feed_filename, *args, **kwargs):
 
 
 def append_feed(schedule, feed_filename, strip_fields=True,
-                chunk_size=5000, agency_id_override=None):
-
+                chunk_size=5000, agency_id_override=None, ignore_failures=True):
```

Comment:
I know that some projects use this library to validate feeds. Historically, pygtfs was very strict. I am fine with adding `ignore_failures` as an option, but I think it should be off by default, and turned on explicitly by users who want it.

Reply:
I can get behind that.